quote_load.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import logging
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("Quote_Load_Logger")
+
+
+class QuoteException(Exception):
+    pass
+
+
+class NoResponseFromQuoteUrl(QuoteException):
+    pass
+
+
+def get_quote(quote_url):
+    try:
+        logger.info('Looking for the inspiring quote...')
+        quote_resp = requests.get(quote_url)
+        quote_text = quote_resp.json()[0]['q']
+        logger.info('Quote text load success')
+        quote_author = quote_resp.json()[0]['a']
+        logger.info('Quote author load success')
+
+        return quote_text, quote_author
+    except requests.exceptions.RequestException as e:
+        logger.info('No response from Quote url')
+        logger.info(e)
+
+        raise NoResponseFromQuoteUrl('No response from Quote url')
+
+    except KeyError as e:

```

Comment:
Same as `KeyError` in [image_load.py](https://github.com/Derector8/Mental-State-Improvement-project/pull/1/commits/765d14069e398c1984dcd1d53486b28b7da90f8c#diff-11e37b8863d0bcb61b2ef871a453a1912dfd6fc62083549e8a3b8a73da6f14d3), consider splitting big try...except block into smaller blocks.